static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -24,40 +36,49 @@ $labour-liverpool-header-element-opacity: .85;
     }
 }
 
-.content__standfirst--labour-liverpool-article {
+.content--labour-liverpool .content__article-body > p:first-of-type {
     @include fs-headline(3);
     color: darken($news-support-4, 25%);
     margin-top: $gs-baseline;
+    font-size: 22px;
+    font-weight: 200;
+}
 
-    .membership-cutout {
-        float: left;
-        width: 140px;
-        margin: 0 $gs-gutter/-2;
-
-        @include mq($from: mobile, $until: tablet) {
-            display: none;
-        }
-    }
+.labour-liverpool-cutout {
+    float: left;
+    width: 140px;
+    margin: 0 $gs-gutter/-2;
 
-    > p:first-of-type {
-        display: inline;
+    @include mq($from: mobile, $until: tablet) {
+        display: none;
     }
 }
 
 .labour-liverpool-quotes__svg {
+    float: left;
     fill: $news-support-4;
     width: 30px;
     height: 18px;
     padding-right: $gs-gutter/4;
 }
 
 .labour-liverpool-headline {
-    position: absolute;
-    top: gs-height(6);
-    max-width: gs-span(8);
+    position: relative;
+    width: 100%;
+    top: $labour-liverpool-headline-top;
```

Comment:
Comments were defined with the variable, admittedly not ideal. I've inlined the comments and the variable